src/main.cpp

```diff
@@ -160,7 +145,7 @@ int main(const int argc, const char *argv[])
     cart_info.write_geojson(
       geo_file_name,
       map_name + "_equal_area",
-      output_to_stdout);
+      false);
```

Comment:
We never want to output the input to stdout. Maybe in the future, we can make this have a default value of false?